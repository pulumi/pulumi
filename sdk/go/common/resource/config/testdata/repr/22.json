{"my:0":"testValue"}