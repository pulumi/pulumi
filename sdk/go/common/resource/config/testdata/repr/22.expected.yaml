decrypt:
  my:0: testValue
paths:
  my:0:
    value: testValue
    string: testValue
    redacted: testValue
    object: FhAABnN0cmluZwwLAAl0ZXN0VmFsdWU=
    secure: false
    isObject: false
